### ISSUE TITLE
re-add fiat balance column to Qt history tab

### ISSUE THREAD

**Issue by SomberNight** [spesmilo/electrum#4209]
A user was complaining on IRC that the latest release does not have a fiat balance column.
It was indeed removed in https://github.com/spesmilo/electrum/commit/0f16bcdc1ff5cda748f3079d7efd983b426277fb

Was the removal deliberate?
We could also make it a (further) preference item.

**Comment by hkop123** [comment]
yes please re-add feature very useful to see usd balance besides balance in bitcoins

**Comment by ecdsa** [comment]
I removed it because it is not really needed in accounting.  what is typically needed is the fiat balance at a given date, e.g. January 1st, and you do not have that infomation with that column.

We can add it back, but I am afraid that users who are asking for it do not really understand what they want. This column does not give you your USD balance, unless you transacted the same day. Your USD balance is shown in the status bar.

**Comment by nibarulabs** [comment]
Correct, it is not needed for accounting, but it is still useful to see what balance * fiat value equals for that line. My team upgraded our wallets recently (yes we were late) and when we found this column gone, it was frowns all around.
